### PR TITLE
Created script to detect unused images

### PR DESCRIPTION
**Pull request by BrookJeynes**
- Created script to detect unused images
- Created github action to use script on PR

image
**Figure: GitHub Action output**

### CODE REVIEW

**Review by bradystroud — CHANGES_REQUESTED** [review]
Looking good 🛶

**Comment by tiagov8** [comment]
Hey @BrookJeynes / @bradystroud 

Should we have a label for these PRs that aren't content edits?

**Comment by bradystroud** [comment]
@tiagov8 @BrookJeynes 
> Should we have a label for these PRs that aren't content edits?

Good idea - done

File: .github/workflows/check-images.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v3
+
+      - name: Check PR
+        id: check_pr
+        run: |
+          echo "is_pr=$(echo ${{ github.event_name }} | tr '[:upper:]' '[:lower:]' | grep -c 'pull_request')" >> $GITHUB_ENV
+
```
Comment (bradystroud) [inline .github/workflows/check-images.yml:31]: Its not clear what this does and what the result is used for

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**